module3/lessons/Week2/FindingYourStrengths.md

```diff
@@ -0,0 +1,153 @@
+
+* Now that brings us to his **talent**, which in this context relates to his ability to engage and connect with others. He’s a connector and educator.
+
+Combining these 3 things leads us to the **strength** known as Inspirational Leadership, which is defined as the ability to uplift, enliven, fill, and empower people with a compelling vision. As the founder of several programs, Jeff has cultivated and used this strength to not only further his vision but get others involved in executing it as well.
+
+</section>
+
+<section class='answer' markdown='1'>
+
+### Example #2
+Let’s look at another strength. A common strength that comes up for students is Persistence, which is defined as the drive to firmly continue in a course of action, despite difficulties, opposition, or warning; stubborn determination.
+
+* A **skill** that would contribute to this is detail-oriented planning. If you’re someone who uses a planner, likes to write to-do lists, and approaches projects from a backwards planning stance, this might describe you!
+
+* The **knowledge** needed here would be around knowing how to prioritize, which comes down to the ability to know which task to focus on at a time based on practice.
+
+* The natural **talent** that might come into play here is the ability to maintain focus during arduous tasks.
+
+Put together, these create the **strength** of persistence.
+
+</section>
+
+## Additional Strategies
+Breaking down strengths in this way can be easier when we already know what the strength is, but how can you approach identifying your strengths on your own?
+
+<section class='answer' markdown='1'>
+### 1. Use the SIGN method
+<section class='note' markdown='1'>
+* Success: Using your strengths makes you feel successful
+* Instinct: Activities you are naturally drawn to
+* Growth: Where you learn the most, come up with new ideas, and have the best insights
+* Needs: Where you feel the need to spend the most time
+</section>
+</section>
+
+
+<section class='answer' markdown='1'>
+### 2. Use others to find your strengths
+<section class='note' markdown='1'>
+* What do others ask for your help with?
+* What do others want you to teach them?
+* What do you receive compliments for?
+</section>
+</section>
+
+<section class='answer' markdown='1'>
+### 3. Use a Strenghs Assessment
+<section class='note' markdown='1'>
+* Use a tool like High 5, StrengthsFinder or Pairin as a starting point
+* Use your own experience to add a narrative to your identified strengths
+</section>
+</section>
+
+## Discussing Our Strengths
+
+<section class='call-to-action' markdown='1'>
+In small groups    
+
+Discuss the results of your High 5 assessment.  Use the following questions as a guide:
```

Comment:
I like that we move into a reflection discussion section here, but I think this part of the lesson would benefit from more structure. 

Reading through the inspiration resources will also take a bit and I think be a helpful resource. 

What about if we start "Discussing Our Strengths" with some individual reflection time where students compile a list of strengths they resonate with both from their High 5 and from reading through the additional resources.

Then we sort of drop a surprise mini-activity on them. Send out lists of groups of 3/4 and have them do a quick brainstorm of strengths for their group mates, 5 minutes total type of things. Then they jump into breakout rooms and each student has a turn where their peers write in the chat the strengths that come to mind for them. Then the student whose turn it is has a reflection question to answer "Are the strengths your peers shared similar to what you listed yourself? Were there any strengths your peers shared that surprised you?"

Then back to individual work time to take the big list of strengths from self-reflection and peers and select top 3 and write/send reflection.

This is just one idea, we could do something else too! But I think more clear structure would be helpful. It might take a while and make this a 1.5 hour PD, but I think would be worth it 🤷

Reply:
I think this is a great activity!  And yes, I think the timing is going to be difficult.  But, I think I'll roll with it (adding this activity) and see how it goes.